### DISCUSSION TITLE
Ability to customize the prompt + preset themes

### DISCUSSION THREAD

**Discussion by DannyBen** [charmbracelet/vhs#46] [Ideas]
I think it would be nice if:

1. There was a setting to customize the prompt (instead of `"> "`, use `"$ "` for example).
2. There were some predefined themes and the `Theme` command would accept either a JSON or a predefined theme name.

**Comment by maaslalani** [comment]
Totally! We'll make this customizable, it will look something like:

```
Set Prompt "$ "
```

**Comment by nocksock** [comment]
I wondered myself, turns out: you can do that already! Simply by modifying the prompt before showing the screen with `Type "PS1='\033[33m$\033[0m '"`

```
Output demo/demo.gif

Set FontSize 32
Set Width 1200
Set Height 600

Hide
Type "PS1='\033[33m$\033[0m '"
Enter
Ctrl+L
Show

Sleep 2
Type "echo 'Hello!'"
Enter
Sleep 2
```

Will result in:

![demo](https://user-images.githubusercontent.com/22164/198392166-55a0a5d3-89d8-44ab-bec5-f1ca4110b756.gif)

Or you could type `zsh`/`fish` to have your system's prompt (or source a bash script to set things up)

**Reply by maaslalani** [reply]
Oh, nice work around @nocksock! Thanks for sharing!

I will eventually add first class support for shell and prompts as settings though so that you can do it in 1 line instead of 4-6 lines!

**Reply by DannyBen** [reply]
Ah! That's a neat trick. For some reason I thought the `>` prompt was not a "real" prompt.
But yes, having a first class `Set` for it would be nice still.
